reporter/src/main/kotlin/reporters/freemarker/FreemarkerTemplateProcessor.kt

```diff
@@ -335,9 +353,30 @@ class FreemarkerTemplateProcessor(
         @Suppress("UNUSED") // This function is used in the templates.
         fun filterForUnresolvedVulnerabilities(vulnerabilities: List<Vulnerability>): List<Vulnerability> =
                 vulnerabilities.filterNot { input.resolutionProvider.isResolved(it) }
+
+        /**
+         * Return a flag indicating that issues have been encountered during the run of an advisor with the given
+         * [capability] with at least the given [severity]. This typically means that the report is incomplete;
+         * therefore, it should contain a corresponding warning.
+         */
+        @JvmOverloads
+        fun hasAdvisorIssues(capability: AdvisorCapability, severity: Severity = Severity.ERROR): Boolean =
```

Comment:
Nit: Consider dropping the default argument for `severity`. IMO "hasAdvisorIssues" reads as if by default *any* issue should be returned, not just errors.